### ISSUE TITLE
Use current path for symlinked files

### ISSUE THREAD

**Issue by globexdesigns** [AtomLinter/linter-eslint#135]
Consider this directory structure:

```
local-js/
local-js/local.js
inner/
inner/.eslintrc
inner/node_modules/
inner/js/
inner/js/inner.js
inner/js/local (symlink to ../../local-js)
inner/js/local/local.js
```

Here is behaviour of Atom for these files:

- `inner/js/inner.js`
 - Correctly uses the /inner/.eslintrc rules!
- `local-js/local.js`
 - Correctly uses default ESLint rules because `/inner/.eslintrc` isn't within scope
- `inner/js/local/local.js`
 - (BUG!) Incorrectly uses default ESLint rules even though `/inner/.eslintrc` is within scope

I suspect this bug occurs because the symlink path is parsed.

Can anything be done on your end to address this? Running `eslint` via command line inside `inner/` properly deals with the symlink and lints `/inner/js/local/local.js`

**Comment by iam4x** [comment]
Command line and this tool are really different, I'll see next week if I have more time to work on open source projects.

**Comment by steelbrain** [comment]
Fixed by v5.0.0
